Scripts/aws_stack_provision.ps1

```diff
@@ -156,17 +154,17 @@ elseif ($EXISTING_INSTANCE_COUNT -eq 0){
       }
       Write-Host "CFN Stack $STACK_NAME is in CREATE_COMPLETE State"
       $PHYSICAL_INSTANCE_ID = ((Get-CFNStackResource -StackName $STACK_NAME -LogicalResourceId INSTANCE).PhysicalResourceId)
-      Write-Host "Trying to fetch VM Password"
+      ## It is recommended to wait 15 minutes before trying to fetch the VM password.
+      Write-Host "Waiting for 15 minutes before trying to fetch the VM password"
+      Start-Sleep -Seconds 900 
```

Comment:
No. Just loop until its succeeds.
And change `Start-Sleep` in function `fetch_vm_password` on line 33 to 20 seconds

Reply:
Updated the wait logic